Formula/glibc.rb

```diff
@@ -152,12 +153,19 @@ def install
 
     # Delete bootstrap binaries after build is finished.
     rm_rf bootstrap_dir
+
+    # Add system ld search paths
+    (prefix/"etc"/"ld.so.conf").atomic_write(ld_so_conf)
   end
 
   def post_install
```

Comment:
I suggest doing
```ruby
(etc/"ld.so.conf").mkpath
```
here in `post_install` as well.

Reply:
`prefix/"etc/ld.so.conf"` will be linked to `etc/"ld.so.conf"`.

Reply:
Sorry, that was a typo. I meant
```ruby
(etc/"ld.so.conf.d").mkpath
```